utils/cli/core/core.py

```diff
@@ -124,14 +124,15 @@ def tasks_create(self, name, labels, overlap, segment_size, bug, resource_type,
             check_url = self.api.git_check(rq_id)
             response = self.session.get(check_url)
             response_json = response.json()
-            log.info('''Awaiting dataset repository for task. Status: {}'''.format(
-                    response_json['status']))
             while response_json['status'] != 'finished':
+                log.info('''Awaiting dataset repository for task. Status: {}'''.format(
+                    response_json['status']))
                 sleep(git_completion_verification_period)
                 response = self.session.get(check_url)
                 response_json = response.json()
-                if response_json['status'] == 'Failed':
+                if response_json['status'] == 'failed':
```

Comment:
I would suggest also adding a check to 'unknown' status. It happens for example when RQ job was deleted and the client requests information about the job. It would also lead to an infinite loop.